docs/src/operations/requirements.md

```diff
@@ -48,11 +48,14 @@ releases at [solanalabs/solana](https://hub.docker.com/r/solanalabs/solana).
 
 ## Software
 
-- We build and run on Ubuntu 20.04.
+- We build and run on Ubuntu 24.04.
 - See [Installing Solana CLI](../cli/install.md) for the current Solana CLI software release.
 
-Prebuilt binaries are available for Linux x86_64 on CPUs supporting AVX2 \(Ubuntu 20.04 recommended\).
-MacOS or WSL users may build from source.
+Prebuilt binaries are currently available targeting x86_64 with AVX2 support on Ubuntu 20.04. We
+will stop publishing these in the near future (as of May 2025).
+[Building from source](https://docs.anza.xyz/cli/install#build-from-source) is required for all
+other supported target platforms (MacOS and Windows) today and suggested for linux as it will soon
```

Comment:
> for all other supported target platforms (MacOS and Windows)

This page is specifically about running a validator. I don't think it is possible to run a validator on Mac or Windows today, but this blurb kind of implies that you can to me.

Also, we post MacOS bins right now but I'm guessing we'll stop posting all bins

Reply:
i'm just dealing with the ubuntu version here. that was wrong before i started

Reply:
> that was wrong before i started

Agreed, but my read was that the new wording has a stronger implication. Idk, maybe "target" implies build but not run